src/partials/3-scanning-page/hero.html

```diff
@@ -1 +1,27 @@
-<h1>scanning-page</h1>
\ No newline at end of file
+<section class="scanning-page">
+    <div class="container">
+        <h1 class="scanning-page__title">3d сканування</h1>
```

Comment:
we have a special class for our text content, use this class please where it need, also in description text content too
<img width="828" alt="Screenshot 2021-08-14 at 23 52 07" src="https://user-images.githubusercontent.com/74025448/129459794-2a3b6db6-8425-46ce-95ab-4494d09d4824.png">